src/xdocs/config_misc.xml

```diff
@@ -1320,6 +1320,128 @@ void foo(String aFooString,
       </subsection>
     </section>
 
+    <section name="OrderedProperties">
+      <subsection name="Description" id="OrderedProperties_Description">
+        <p>Since Checkstyle 8.21</p>
+        <p>
+          Detects if keys in properties files are in correct order.
+        </p>
+        <p>
+          Rationale: Unsorted property makes merges more easy. While there are no errors in
```

Comment:
> Unsorted property makes merges more easy. While there are no errors in runtime.

I do not understand this. 

Did you mean "Sorted properties make it easy for human to find required property by name in file."

Reply:
Done.